esp/bindings/http/platform/httpservice.cpp

```diff
@@ -377,18 +377,24 @@ int CEspHttpServer::processRequest()
 
             checkSetCORSAllowOrigin(thebinding, m_request, m_response);
 
-            if (thebinding!=NULL)
+            if (thebinding && thebinding->isUnrestrictedSSType(stype))
```

Comment:
Moved this earlier so that the span could be created whether or not thebinding is set.